### DISCUSSION TITLE
Consulta sobre visualización de nombres de calles en tamaño A0

### DISCUSSION THREAD

**Discussion by hfsoriad** [qgispe/discusiones#5] [Preguntas&Respuestas] [answered]
Hola a todos. Tengo una duda al respecto. ¿Por qué en el compositor del mapa, con escala de trabajo de 1:14000 en tamaño A0, el nombre de las calles se visualizan de forma desordenada y grande? Se supone que todo debería corregirse cuando se altera el tamaño de letra y dándole prioridad alta, Adjunto las imágenes.

1. En el área de trabajo.
![1](https://user-images.githubusercontent.com/25108859/119246352-85185000-bb46-11eb-8c8a-324d47d0efc7.jpg)

2. Visualización de los nombres a escala 1:1 en el compositor de imprensión
![2](https://user-images.githubusercontent.com/25108859/119246371-a6793c00-bb46-11eb-8fe4-35f7dbf23a4c.jpg)

3. Captura en tamaño A0
![3](https://user-images.githubusercontent.com/25108859/119246389-c872be80-bb46-11eb-9965-604c47218880.jpg)

Adjunto el archivo del proyecto: https://drive.google.com/file/d/14_Y3wd8l_19pL_E9zBwJlZtW9NYBOMCN/view?usp=sharing

PD: El tamaño de fuente es 1,5 relacionado al nombre de las calles, pero no se consigue visualizar correctamente en escala 1:1, ya que si lo aumento el tamaño (por ejemplo >= 2), sale horizonatal, desordenado y grande, sin seguir la orientación correcta de las calles.

Estaré agradecido por su respuesta y atención.

Saludos.

Henry

**Comment by Fertuesta25** [comment, marked as answer]
@hfsoriad seria excelente si nos compartes tu proyecto para poder ayudarte con algunas pruebas.

**Reply by hfsoriad** [reply]
@Fertuesta25 Ya fue actualizado el mensaje. Los datos están comprimidos en un archivo rar. Me envías una solicitud para darte acceso a ello. Gracias.
